docs/developer/checkout/cookbook.mdx

```diff
@@ -98,3 +98,95 @@ mutation {
+
+Remaining amount:
+- Checkout passes the remaining `amount` to the [`transactionInitialize`](/api-reference/payments/mutations/transaction-initialize.mdx) mutation and desired action such as `CHARGE`.
+- In the `TRANSACTION_INITIALIZE_SESSION` webhook, the payment app validates the split (e.g., if it is allowed) and includes `CHARGE` in [allowed actions](/developer/extending/webhooks/synchronous-events/transaction#response-4).
+
+2. **Process transactions:**
+ - Calling [`transactionProcess`](/api-reference/payments/mutations/transaction-process.mdx) with `action` set to `AUTHORIZE` for the first transaction and `CHARGE` for the second transaction.
+
+:::note
+While it is possible to call `transactionInitialize` and `transactionProcess` directly from the storefront (client-side), it is recommended that these operations be executed from the backend (server-side), which would be more resilient and maintainable.
+:::
+
+## Product personalization
+
+Common use cases of product personalization are:
+
+- Customized products (e.g., engraved jewelry, custom t-shirts)
+- Packaging preferences
+- Product configuration such as PC components, furniture, cars, etc.
+- Delivery preferences for each item
+
+The personalization can be achieved by adding additional [`metadfields`](/api-reference/checkout/objects/checkout-line.mdx#checkoutlinemetafieldstring--) fields to the [`CheckoutLine`](/api-reference/checkout/objects/checkout-line.mdx) object.
+
+With the following mutations:
+- [`checkoutLinesAdd`](/api-reference/checkout/mutations/checkout-lines-add.mdx)
+- [`checkoutLinesUpdate`](/api-reference/checkout/mutations/checkout-lines-update.mdx)
+
+The [`metdafields`](/api-reference/orders/objects/order-line.mdx#orderlinemetafieldstring--) will be copied to the [`OrderLine`](/api-reference/orders/objects/order-line.mdx) after checkout completion.
+
+Additional steps might be required to process such fields in the fulfillment process, such as:
+
+- Pass the fields to the ERP system
+- Listen to [`webhook`](/developer/extending/webhooks/overview.mdx) events such as `ORDER_CREATED` to process the fields
+- [Custom pricing](#custom-pricing)
+
+## Custom product pricing
+
+To set prices on checkout lines dynamically you can use the [`checkoutLinesUpdate`](/api-reference/checkout/inputs/checkout-line-update-input.mdx#checkoutlineupdateinputpricepositivedecimal--) mutation.
+See example [repository](https://github.com/saleor/saleor-app-checkout-prices) for creating custom pricing middleware.
+
+
+## Using phone number instead as identity
+
+In some cases, you might want to use a phone number instead of an email address for customer identification. While Saleor always requires email to create orders, you can email field as a phone number or other semantic meaning. For example: `123456789@phone`.
```

Comment:
```suggestion
In some cases, you might want to use a phone number instead of an email address for customer identification. While Saleor always requires email to create orders, you can email field as a phone number or other semantic meaning. For example: `123456789@noreply.yourcompany.com`.
```